dll/ntdll/ldr/ldrapi.c

```diff
@@ -1000,6 +1000,8 @@ LdrQueryProcessModuleInformationEx(
     ANSI_STRING AnsiString;
     PCHAR p;
 
+    UNREFERENCED_PARAMETER(ProcessId);
```

Comment:
This is an unimplemented code path, so it makes no sense to mark this as `UNREFERENCED_PARAMETER`

Reply:
Print "Unimplement" instead seems better